core/src/test/java/org/jdbi/v3/core/extension/TestDefaultExtensionMethods.java

```diff
@@ -103,7 +103,7 @@ default void finalize() {}
         class Impl implements ExtensionHandler {
 
             @Override
-            public Object invoke(HandleSupplier handleSupplier, Object target, Object[] args) {
+            public Object invoke(HandleSupplier handleSupplier, Object target, Object... args) {
```

Comment:
Kinda weird to use varargs here, since it is an "invocation handler" style which you would rarely / never call and expect varargs collection

Reply:
the overridden parent uses varargs.